conn.go

```diff
@@ -140,7 +137,6 @@ func getNeoBase(url string) (*neo4jBase, error) {
 
 	neoBase := neo4jBase{}
 	err = json.NewDecoder(res.Body).Decode(&neoBase)
-	io.Copy(ioutil.Discard, res.Body)
```

Comment:
These are strange.  This is not necessary and I believe potentially dangerous.  It's at best inefficient, so I removed them, but at worst it could be causing our issues.